Apps/Sandcastle/gallery/Terrain.html

```diff
@@ -34,16 +34,19 @@
 //Sandcastle_Begin
 var viewer = new Cesium.Viewer('cesiumContainer');
 
-var scene = viewer.scene;
-var globe = scene.globe;
-globe.depthTestAgainstTerrain = true;
-
 var cesiumTerrainProviderMeshes = new Cesium.CesiumTerrainProvider({
     url : '//assets.agi.com/stk-terrain/world',
     requestWaterMask : true,
     requestVertexNormals : true
 });
+viewer.terrainProvider = cesiumTerrainProviderMeshes;
+
+var mountEverest = new Cesium.Cartesian3(300770.50872389384, 5634912.131394585, 2978152.2865545116);
```

Comment:
@mramato You have access to the fork if you'd like to add the sandcastle changes.